control-plane/roles/monitoring/README.md

```diff
@@ -41,6 +41,9 @@ The following variables can be set to configure the role:
 | monitoring_grafana_additional_datasources     |           | Configures additional datasources for Grafana                                                                                                                                                                                   |
 | monitoring_grafana_github_oauth               |           | [Configure GitHub OAuth2 authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/github/)                                                                             |
 | monitoring_grafana_extra_secret_mounts        |           | Extra secret mounts                                                                                                                                                                                                             |
+| monitoring_custom_dashboards_enabled          |           | Enables adding custom dashboards to grafana.                                                                                |
+| monitoring_custom_dashboard_folder            |           | Folder where the custom dashboards are located in the deployment.                                                               |
```

Comment:
I think just providing the file list should be enough? People can put this into their `files` folder and just use:

```yaml
monitoring_custom_dashboard_file_list:
  - dashboard1.j2
``` 

Ansible will then pick them up correctly. If they are in a subfolder the list would look like:

```yaml
monitoring_custom_dashboard_file_list:
  - custom_dashboards/dashboard1.j2  
  - custom_dashboards/dashboard2.j2
``` 

Maybe also add these as an example to the `defaults` (commented).

Then you also do not have to use `monitoring_custom_dashboards_enabled` anymore. Just when there are files given, iterate over them and deploy.